framework/tlvf/test/tlvf_test.cpp

```diff
@@ -189,6 +190,88 @@ int test_complex_list()
     return errors;
 }
 
+bool add_encrypted_settings(std::shared_ptr<tlvWscM2> m2, uint8_t *keywrapkey, uint8_t *iv)
```

Comment:
This function contains a copy of son::master_thread::autoconfig_wsc_add_m2_encrypted_settings(). That doesn't make much sense, because we actually want to test that function. So to make it testable, it should be moved out somehow.

Reply:
Perhaps we can move it to they tlvf none generated files?

Reply:
will do that in a separate PR / commit, keeping discussion open so I don't forget.